### DISCUSSION TITLE
Question: Passthrough of blockdevice?

### DISCUSSION THREAD

**Discussion by Norcoen** [vagrant-libvirt/vagrant-libvirt#1375] [General]
Maybe I'm blind, but how do I pass a blockdevice to the vm?
In my case I build images for raspberry pi and want to write them to `/dev/mmcblk0` directly, so I wanted to make it accessible in the vm directly.
I have done this before with virt-manager, so it works, I just cant figure out how how to configure vagrant-libvirt for it.

**Comment by electrofelix** [comment]
We're missing support to pass through block devices, however you might be able to use a PCI device pass through solution in this case https://github.com/vagrant-libvirt/vagrant-libvirt#pci-device-passthrough

Alternatively you can make use of qemuargs to pass additional options to the VM if you know the right combination, you can see an example of this being used here https://github.com/vagrant-libvirt/vagrant-libvirt/issues/1250#issuecomment-818623208
